### PR TITLE
Media card design and code improvements

### PR DESCRIPTION
**Pull request by Tharon-C**
This PR make some design changes to MediaCard based on work done to redesign The InstanceLaunchModal. Also, some a bug was discovered at this time where react-dimensions renders a list of zeros if the initial element doesn't have a height or width. There are hacky solutions to this bug but the benefit doesn't justify stooping so low so it was removed. I'll be looking for a better solution in the meantime mediaqueries get it done./

* Removes react-dimensions
  * Was causing a bug that had workarounds but the ends didn't justify the means
* Adds an ActiveLink prop and render
  * This is the same as QuickLinks but always shows where QuickLinks show on hover or if open.
* Design changes
  * Add left border when open
  * Less vertical margin when open
  * Title section is wider

### Future PR work
* Rename `QuickLinks` and `ActiveLinks` to `QuickActions` and `ActiveActions`
  * These are for the group of actions that appear to the right of MediaCards. Open to feedback on naming, the term "link" was an artifact of my excessive misuse of link elements to make buttons in a past life. I only want to change it once and do it in a very deliberate way since it is an API change.  
  
### MediaCards after changes 
Shows the bookmark is added to `activeLinks` when checked, the vertical margins and border.

ol4gxeldcf

### CODE REVIEW

File: src/MediaCard.js [hunk 1]
```diff
@@ -86,7 +109,7 @@ const MediaCard = React.createClass({
                 />
             )
```
Comment (lenards) [inline src/MediaCard.js]: Missing semi-colon here.

Comment (lenards) [reply src/MediaCard.js]: That was existing code, so /shrug.

**Review by lenards — APPROVED** [review]

**Comment by lenards** [comment]
:shipit:

**Comment by Tharon-C** [comment]
Added missing semicolon, and rebased. Thanks for reviewing @lenards! 
:shipit:

**Pull request merged**